src/app/pages/download/download.twig

```diff
@@ -9,7 +9,14 @@
     <header>
       <h1>
         {{ headerTitle }}
-        <div  title="Latest Release Date" style="float: right; font-size: 18px;">{{ releaseDate }}</div>
+        <span class="download-infos">
```

Comment:
All the right aligned stuff is not very responsive.
![image](https://user-images.githubusercontent.com/7498380/147448183-ec7e616d-87e6-47e6-b2db-13a9f0ee58fa.png)

Reply:
I improved it a little bit, but yes I'm also not very pleased with the current result. But in general what do you think about this release notes link below the release date?